### ISSUE TITLE
next/image generates invalid html tag (empty alt attribute)

### ISSUE THREAD

**Issue by meliborn** [vercel/next.js#27163]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

15.7.0

### What browser are you using?

safari

### What operating system are you using?

macOS

### How are you deploying your application?

other

### Describe the Bug

As I see, `next/image` generates some divs with image inside it. I don't know the exact reason for that, but this additional image has empty `alt` attribute which prevent site from being passed html validator.

![image](https://user-images.githubusercontent.com/735194/125597894-73c5d81b-4b25-4fe3-a8f4-f5be7d79a2d9.png)

![Снимок экрана 2021-07-14 в 12 23 01](https://user-images.githubusercontent.com/735194/125597938-aeb54a35-a1b7-4f97-9dd8-a71848b58905.png)

### Expected Behavior

To pass validation

### To Reproduce

```javascript
import Image from 'next/image';

<Image
	{...rest}
	src={(imgSrc as StaticImageData) || fallback}
	alt={alt}
	unoptimized
	className={className}
	onError={() => {
		setImgSrc(fallback);
	}}
/>
`

**Comment by Eliepse** [comment]
This image is generated by Next.js as a placeholder until the actual intended is loaded.
The problem here is that the role should not be present as the image is already ignored by screen readers or such.
I think the expected behaviour is that the generated image should not have a role at all since the alt attibute is empty, and aria-hidden is true.

**Comment by styfle** [comment]
Agreed, the issue is the role 👍 

See PR #27767 for the fix, thanks!

**Comment by Eliepse** [comment]
Thank you!

**Comment by styfle** [comment]
This was released in [v11.0.2-canary.26](https://github.com/vercel/next.js/releases/tag/v11.0.2-canary.26).

You can try it out today with `yarn add next@canary`, thanks!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
